docs/airflow/production-maintenance.md

```diff
@@ -40,8 +40,7 @@ DAGs are listed in alphabetical order, as they appear in the Airflow UI.
 `gtfs_schedule_history2` | Yes | **📆 Yes** | **🔂 No** | `gtfs_ loader` | |
 `parse_and_validate_rt` | Yes | No | **🔂 No** | N/A | |
 `payments_loader` | Yes | No | Yes | N/A | |
-`payments_views` | Yes | No | Yes | `payments_ views_ staging`| |
-`payments_views_staging` | Yes | No | Yes | `payments_ loader` | |
+`payments_views` | Yes | No | Yes | | |
```

Comment:
Tiniest nitpick -- technically for consistency we'd want an N/A in the `depends on` column here (rather than blank)